server/controllers/bookController.js

```diff
@@ -90,6 +90,50 @@ class BookController {
       Utils.errorStatus(res, 500, error.message);
     }
   }
+
+  static async lendBook(req, res) {
+    const { isbn, title, patronId, cost} = req.body;
```

Comment:
- You should be validating if the patron actually exists in the database and also reference the user's table.

- I don't think you need the title because you are only checking with the ISBN. 

- And you should validate the ISBN too if they exist on the database